Java Solutions/Day24.java

```diff
@@ -1,17 +1,26 @@
 
 public class Day24 {
 
-	static Node removeDuplicates(Node head) {
-        Node current = head;
-        
-        while (current != null && current.next != null) {
-            while (current.next != null && current.data == current.next.data){
-                current.next = current.next.next;
-            }
-            current = current.next;
-        }
+public static Node removeDuplicates(Node head) {
```

Comment:
This is a good approach that also solves the problem, but I'm not sure if it is much more efficient than the first approach. It is recursive instead of iterative, which makes it use O(n) memory for the stack of calls.
Maybe we can keep both methods, rename the first one to removeDuplicatesIterative() and the second one to removeDuplicatesRecursive(), what do you think?